eventhubs/spring-cloud-azure-stream-binder-eventhubs/eventhubs-binder/README.md

```diff
@@ -1,210 +1,164 @@
+```shell
+az account set --subscription <your-subscription-id>
+```
+
+### Provision the Resources
+
+After login Azure CLI with your account, now you can use the terraform script to create Azure Resources.
+
+```shell
+# In the root directory of the sample
+# Initialize your Terraform configuration
+terraform -chdir=./terraform init
+
+# Apply your Terraform Configuration
+# Type `yes` at the confirmation prompt to proceed.
+terraform -chdir=./terraform apply
+
+```
+
+
+
+
+It may take a few minutes to run the script. After successful running, you will see prompt information like below:
+
+```shell
+
+
+azurerm_resource_group.main: Creating...
+azurerm_resource_group.main: Creation complete after 3s [id=/subscriptions/799c12ba-353c-44a1-883d-84808ebb2216/resourceGroups/rg-eventhubs-binder-nxatj]
+azurerm_eventhub_namespace.eventhubs_namespace: Creating...
+azurerm_storage_account.storage_account: Creating...
+...
+azurerm_storage_account.storage_account: Creation complete ...
+azurerm_storage_container.storage_container: Creating...
+azurerm_role_assignment.role_storage_account_contributor: Creating...
+azurerm_storage_container.storage_container: Creation complete ...
+azurerm_role_assignment.role_storage_blob_data_owner: Creating...
+...
+azurerm_role_assignment.role_storage_blob_data_owner: Creation complete ...
+azurerm_role_assignment.role_storage_account_contributor: Creation complete ...
+...
+azurerm_eventhub_namespace.eventhubs_namespace: Creation complete ...
+azurerm_eventhub.eventhubs: Creating...
+azurerm_eventhub.eventhubs: Creation complete ...
+...
+azurerm_role_assignment.role_eventhubs_data_owner: Creation complete ...
+
+Apply complete! Resources: 8 added, 0 changed, 0 destroyed.
+
+Outputs:
+...
+
+
+```
+
+You can go to [Azure portal](https://ms.portal.azure.com/) in your web browser to check the resources you created.
+
+### Export Output to Your Local Environment
+Running the command below to export environment values:
+
+```shell
+ source ./terraform/setup_env.sh
+```
+
+## Run Locally
+
+In your terminal, run `mvn clean spring-boot:run`.
+
+
+```shell
+mvn clean spring-boot:run
+```
+
+## Verify This Sample
+
+1.  Verify in your app’s logs that similar messages were posted:
+
+```shell
+New message received: 'Hello world, 17' ...
+Message 'Hello world, 17' successfully checkpointed
+...
+New message received: 'Hello world, 18' ...
+Message 'Hello world, 18' successfully checkpointed
+...
+New message received: 'Hello world, 27' ...
+Message 'Hello world, 27' successfully checkpointed
+
+```
+
+
+## Clean Up Resources
+After running the sample, if you don't want to run the sample, remember to destroy the Azure resources you created to avoid unnecessary billing.
+
+The terraform destroy command terminates resources managed by your Terraform project.   
+To destroy the resources you created.
+
+```shell
+terraform -chdir=./terraform destroy
+```
+
 
 ## Enhancement
```

Comment:
Do we need the Enhancement here?

Reply:
It's in the original doc, not sure, let me ask @yiliuTo for help.

Reply:
Talked with Yi with Teams, I think we can keep this.

Reply:
@backwind1233  @yiliuTo  do you think we should have an extended step for all of our samples, and link them to the reference doc perhaps, maybe we should put this enhancement section into the reference doc?

Reply:
Both for me are ok, how do you think about it @yiliuTo.

Reply:
I agree that ref docs should have enhancement introduction, and as for the samples, referring to other open source sample projects, like scs, gcp,, they split each sample project in the way that each sample only introduces one feature, and then the readme could have a short introduction about that feature. Perhaps we could consider this option in the future.
And for now, I agree with @saragluna 's suggestion to put the enhancement in ref docs and a link of it in the samples.

Reply:
Agree, I will add an issue to track this, but deleted it in the readme files at this moment. 
After the Enhancement part is added in the reference docs, I will paste the link in the sample readme files.